src/MLE.jl

```diff
@@ -146,7 +146,7 @@ end
 
 function fit!(gas::Model{D, T}, y::Vector{T};
               initial_params::Matrix{T} = DEFAULT_INITIAL_PARAM,
-              opt_method::AbstractOptimizationMethod = LBFGS(gas, DEFAULT_NUM_SEEDS),
+              opt_method::AbstractOptimizationMethod = NelderMead(gas, DEFAULT_NUM_SEEDS),
```

Comment:
Why are we changing the default to `NelderMead`?

Reply:
in my head it was already, this was kind of a bug, I changed in a place and not the other.